framework/wazuh/cluster/cluster.py

```diff
@@ -117,12 +114,15 @@ def read_config():
     for value_name in set(cluster_default_configuration.keys()) - set(config_cluster.keys()):
         config_cluster[value_name] = cluster_default_configuration[value_name]
 
+    if isinstance(config_cluster['port'], str) and not config_cluster['port'].isdigit():
+        raise WazuhException(3004, "Cluster port must be an integer.")
+
     config_cluster['port'] = int(config_cluster['port'])
     if config_cluster['disabled'] == 'no':
         config_cluster['disabled'] = False
     elif config_cluster['disabled'] == 'yes':
         config_cluster['disabled'] = True
-    else:
+    elif not isinstance(config_cluster['disabled'], bool):
         raise WazuhException(3004, "Allowed values for 'disabled' field are 'yes' and 'no'. Found: '{}'".format(
             config_cluster['disabled']))
 
```

Comment:
What is all this commented code?

Reply:
This code was added in https://github.com/wazuh/wazuh/pull/2238. It wasn't correctly working so it was commented until we could take a deeper look into it.